lib/link.ts

```diff
@@ -70,14 +78,14 @@ export const handleLink = async (url: string) => {
       const callback = decodeURIComponent(rawCallback);
 
       console.info("Navigating to NWA flow");
-      router.push({
+      safeRouterPush({
```

Comment:
I'm getting an error: "The action 'POP_TO_TOP' was not handled by any navigator" (but the navigation still works)

Reply:
Fixed now, this was happening because we call `router.dismissAll` although this error doesn't happen in android. I also changed to not use handleLink in send so we now replace instead of push, just like we do it on scanning a lightning address QR